### DISCUSSION TITLE
Displaying a sponsor button for your organization

### DISCUSSION THREAD

**Discussion by wols** [FossifyOrg/General-Discussion#24] [Ideas]
[GitHub : Setting up GitHub Sponsors for your organization](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-organization)

Use the money e.g. for registering the label "Fossify" ;-)

**Comment by naveensingh** [comment]
@wols thanks, I have already applied for approval :)
It'll probably take a few days for the Github staff to process it.

**Reply by wols** [reply]
Good luck, this relates to #25

**Reply by inson1** [reply]
@wols No it doesnt :) You dont need to be non profit org for that.

**Comment by naveensingh** [comment]
@wols GitHub Sponsor profile is now live: https://github.com/sponsors/FossifyOrg
